packages/workbox-strategies/_default.mjs

```diff
@@ -56,6 +58,15 @@ const mapping = {
 const defaultExport = {};
 Object.keys(mapping).forEach((keyName) => {
   defaultExport[keyName] = (options = {}) => {
+    if (process.env.NODE_ENV !== 'production') {
+      if (options.cacheExpiration && !options.cacheName) {
```

Comment:
This isn't needed anymore as the cacheExpiration option doesn't get used - developers have to explicitly create the Plugin.